src/otx/recipe/_base_/data/semisl/semantic_segmentation_semisl.yaml

```diff
@@ -0,0 +1,147 @@
+task: SEMANTIC_SEGMENTATION
+input_size:
+  - 512
+  - 512
+mem_cache_size: 1GB
+mem_cache_img_max_size: null
+image_color_channel: RGB
+data_format: common_semantic_segmentation_with_subset_dirs
+include_polygons: true
+unannotated_items_ratio: 0.0
+ignore_index: 255
+train_subset:
+  subset_name: train
+  batch_size: 8
+  num_workers: 4
+  transform_lib_type: TORCHVISION
+  to_tv_image: true
```

Comment:
As far as I know, the Transform that exists in otx can take input as a numpy array, so we don't need to replace it with `tv_image`. So `False` seems appropriate for this, could you please confirm?

As it is, expect one more unnecessary transformation to happen: numpy array -> tv_image -> numpy array.